js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -1,5 +1,5 @@
 function alternatingCharacters(arrOfStrings) {
-  var alternatingCharactersCount = []
+  var alternatingCharactersCount = [];
```

Comment:
use `const` wherever possible by default.
if the variable is going to be reassigned use let.
also please use eslint for linting.